playbooks/common/openshift-cluster/upgrades/rpm_upgrade.yml

```diff
@@ -5,7 +5,3 @@
 - name: Ensure python-yaml present for config upgrade
   action: "{{ ansible_pkg_mgr }} name=PyYAML state=present"
   when: not openshift.common.is_atomic | bool
```

Comment:
I'd rather keep things in line with the majority of the release-1.3 branch which uses this pattern. The only place we've used package module in this branch is when we backported conntrack installation from master.
Service restarts are handled later in the playbook.